webhooks/virtualmachine/validation/virtualmachine_validator.go

```diff
@@ -2251,10 +2251,9 @@ func (v validator) validateVMAffinity(
 	if a := affinity.VMAntiAffinity; a != nil {
 		p := path.Child("vmAntiAffinity")
 
-		// TODO(sai): Allow this for privileged users.
-		if len(a.RequiredDuringSchedulingPreferredDuringExecution) > 0 {
-			allErrs = append(allErrs, field.Forbidden(p.Child("requiredDuringSchedulingPreferredDuringExecution"),
-				"VM anti-affinity with RequiredDuringSchedulingPreferredDuringExecution is not allowed"))
+		// Mobility Operator needs to specify RequireDuringScheduling type policies for VM-VM Host AAF.
```

Comment:
One of the AIs from the code walkthrough was that we will allow both "required" and "preferred" during scheduling variants for VM-VM zonal anti-affinity.  Can you handle that as well?